### ISSUE TITLE
Unhandled rejection undefined

### ISSUE THREAD

**Issue by Proantagonist** [knightzac19/NodeJS-ArkData#1]
I'm trying to get the data server to work. I'm not very experienced with NodeJS so I would really like any help you can issue. 

The output is as follows:

Config verification finished!
Initializing Player Data...
Config verification finished!
Doesn't look like bans are in the Linux Folder, going to try the Windows location...
Caching Steam Info...
Profiles are done updating!
Steam bans are done updating!
Unhandled rejection undefined
Initializing Player Data...
Initializing Player Data...
Doesn't look like bans are in the Linux Folder, going to try the Windows location...
Doesn't look like bans are in the Linux Folder, going to try the Windows location...
Trace: saveplayer error!
    at Statement.<anonymous> (D:\Github\NodeJS-ArkService\node_modules\arkdata\lib\playercache.js:74:25)
    at Statement.replacement (D:\Github\NodeJS-ArkService\node_modules\sqlite3\lib\trace.js:19:31)
events.js:160
      throw er; // Unhandled 'error' event
      ^

Unhandled rejection undefined is coming from bluebird to the best of my knowledge.

Please let me know what other information you required to point me in the right direction. 

Thank you!

**Comment by knightzac19** [comment]
Can you check your settings.json and make sure it's pointing to the right location? and also reply with your nodejs version and OS you're on.

Thanks.

**Comment by Proantagonist** [comment]
Thank you for your reply.

In the settings.json:

 "ark_path": "C:\\\Ark_Server\\\Base"

Where "C:\Ark_Server\Base" contains:
 - Engine
 - ShooterGame
 - steamapps
 - some dll's, version.txt, PackageInfo.bin, and SteamCMDInstall.sh

Current nodejs version is 7.4.0 and have tested on Windows 10, Xubuntu 14, and OSX Sierra.

Thank you.

**Comment by knightzac19** [comment]
It looks like that's when it's trying to save the players to the sqlite3 database, can you make sure that players.sqlite is getting created. I made a couple updates to the file that you were having trouble with. Run npm update and let me know if it at least shows you the error.

**Comment by Proantagonist** [comment]
Oh! Not sure if it was your changes or something else but I actually installed a fresh copy of the project, changed the variable values, and it worked first try! Thank you!

Now I am having another issue however, when I'm making the REST calls, particularly to the listOnline endpoint, the response is very sporadic. All the others seem to be working OK from what I've tested. I'm using angular's $http service but I got the same result with your  JQuery snippet. Should I create an issue on that repo and further explain?

Edit: may have something to do with cross-origin policies. Seems to be working well while running everything within the server but not if I allow "*" in the "allowed_server" instead of specifying just one.

Thank you for your time. Can I buy you a drink?
